bin/peas_shell

```diff
@@ -164,6 +165,11 @@ class PanSensorShell(cmd.Cmd):
         self.weather = AAGCloudSensor(serial_address=port, store_result=True)
         self.do_enable_sensor('weather')
 
+    def clear_buffer(self):
```

Comment:
This appears to be unused.

Reply:
Thanks! @jamessynge 
clear_buffer will help to clean both i/p o/p buffers of the assigned port, i am not sure where it can be helpful.
Obviously it will be required before reading / writing something in to the port but i am not sure where in peas_shell.
It will require a bit discussion.

Reply:
I would suggest making it a callable option, i.e name it `do_clear_buffer`.

It should then be made more generic, so it can be resetting either `environment` or `weather` via `self.active_sensors[sensor].ser.reset_input_buffer()`. There are other example in the file of handling the params passed by the user.